Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Core/SignContract/Signatures/Signature.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.Contracts.Core.SignContract.Signatures;
+
+using System.Text.RegularExpressions;
+using Exceptions;
```

Comment:
You can move both usings to Global class

Reply:
Ok